plugins/woocommerce-blocks/assets/js/base/components/quantity-selector/test/index.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { render, screen } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import QuantitySelector, { QuantitySelectorProps } from '../index';
+
+const defaults = {
+	disabled: false,
+	editable: true,
+	itemName: 'product',
+	maximum: 9999,
+	onChange: () => void 0,
+} as QuantitySelectorProps;
+
+describe( 'QuantitySelector', () => {
+	it( 'The quantity step buttons are rendered when the quantity is editable', () => {
+		render( <QuantitySelector { ...defaults } /> );
+
+		expect(
+			screen.getByLabelText(
+				`Increase quantity of ${ defaults.itemName }`
+			)
+		).toBeInTheDocument();
+		expect(
+			screen.getByLabelText( `Reduce quantity of ${ defaults.itemName }` )
+		).toBeInTheDocument();
+	} );
+
+	it( 'The quantity step buttons are not rendered when the quantity is not editable', () => {
+		render( <QuantitySelector { ...defaults } editable={ false } /> );
+
+		expect(
+			screen.queryByLabelText(
+				`Increase quantity of ${ defaults.itemName }`
+			)
+		).not.toBeInTheDocument();
+		expect(
+			screen.queryByLabelText(
+				`Reduce quantity of ${ defaults.itemName }`
+			)
+		).not.toBeInTheDocument();
+	} );
+} );
```

Comment:
This is great, thanks a lot for adding these tests. 

Just a note, I would have used `rerender` from the result of the first `render` and changed the props, like so:

```tsx
		const { rerender } = render( <QuantitySelector { ...defaults } /> );
```

then instead of creating a second test for this, I'd have re-rendered the component like so

```tsx
		rerender( <QuantitySelector { ...defaults } editable={ false } /> );
```
therefore keeping this test to just one spec. Totally up to you if you want to change it though, I don't feel strongly about it, just providing an alternate approach.

Reply:
No problem for me boss, it's your codebase :sweat_smile: adjusted with rerender :+1: